Reportedly/Modules/Authentication/Signup/SignupPresenter.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  SignupPresenter.swift
+//  Reportedly
+//
+//  Created by Mikey Pham on 12/1/20.
+//  Copyright © 2020 NimbleHQ. All rights reserved.
+//
+
+final class SignupPresenter {
+
+    let router: SignupRouterInput
+    let interactor: SignupInteractorInput
```

Comment:
Please set private these inputs ✌️

Reply:
oh they are generated from the template, sure let me update 🙏